### PR TITLE
Add ability co configure the script by ENV var for issue #5

### PR DESCRIPTION
**Pull request by jmaitrehenry**
I use this script on a docker container, it's really helpfull to be able to configure it via nev.

### CODE REVIEW

**Comment by jmaitrehenry** [comment]
Fix #4

File: README.md [hunk 1]
```diff
@@ -17,6 +17,17 @@ optional arguments:
                         Config file location
   -1, --once        Run once and exit
 ```
+Configuration via environnement
```
Comment (briancline) [inline README.md]: Minor spelling fix needed here for "environment"

**Comment by briancline** [comment]
Looks good to me, just one minor thing I noted in the readme. Thanks for the patch and for including documentation.

What do you think @sudorandom?

**Comment by sudorandom** [comment]
Looks good to me after the typo fix, +1

**Comment by jmaitrehenry** [comment]
I just fix the typo, thanks!

**Pull request merged**